includes/migration-guide/retargeting/jit/new-64-bit-jit-compiler-net-framework-46.md

```diff
@@ -3,7 +3,7 @@
+|Suggestion|<strong>Mitigation of known issues</strong> If you encounter the issues listed above, you can address them by doing any of the following:<ul><li>Upgrade to the .NET Framework 4.6.2. The new 64-bit compiler included with the .NET Framework 4.6.2 addresses each of these known issues.</li><li>Ensure that your version of Windows is up to date by running Windows Update. Service updates to the .NET Framework 4.6 and 4.6.1 address each of these issues except the <xref:System.NullReferenceException> in an unboxing operation.</li><li>Compile with the older 64-bit JIT compiler. See the <strong>Mitigation of other issues</strong> section for more information on how to do this.</li></ul><strong>Mitigation of other issues</strong> If you encounter any other difference in behavior between code compiled with the older 64-bit compiler and the new 64-bit JIT compiler, or between the debug and release versions of your app that are both compiled with the new 64-bit JIT compiler, you can do the following to compile your app with the older 64-bit JIT compiler:<ul><li>On a per-application basis, you can add the [<](~/docs/framework/configure-apps/file-schema/runtime/uselegacyjit-element.md) element to your application's configuration file. The following disables compilation with the new 64-bit JIT compiler and instead uses the legacy 64-bit JIT compiler.</li></ul><pre><code class="language-xml">&lt;?xml version =&quot;1.0&quot;?&gt;&#13;&#10;&lt;configuration&gt;&#13;&#10;&lt;runtime&gt;&#13;&#10;&lt;useLegacyJit enabled=&quot;1&quot; /&gt;&#13;&#10;&lt;/runtime&gt;&#13;&#10;&lt;/configuration&gt;&#13;&#10;</code></pre><ul><li>On a per-user basis, you can add a <code>REG_DWORD</code> value named <code>useLegacyJit</code> to the <code>HKEY_CURRENT_USER\SOFTWARE\Microsoft\.NETFramework</code> key of the registry. A value of 1 enables the legacy 64-bit JIT compiler; a value of 0 disables it and enables the new 64-bit JIT compiler.</li><li>On a per-machine basis, you can add a <code>REG_DWORD</code> value named <code>useLegacyJit</code> to the <code>HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework</code> key of the registry. A value of <code>1</code> enables the legacy 64-bit JIT compiler; a value of <code>0</code> disables it and enables the new 64-bit JIT compiler.</li></ul>You can also let us know about the problem by reporting a bug on [Microsoft Connect](https://connect.microsoft.com/VisualStudio).|
```

Comment:
Again unrelated to your PR, on the ninth line from the top, could you change the [\<] to [\\\<useLegacyJit>]? The link has dropped out of the build.